src/main/java/co/inventorsoft/StreamHomework.java

```diff
@@ -29,7 +31,10 @@ public List<Person> extractTeenagers(final List<Person> people) {
      * @return collection of user, without duplicates
      */
     public List<User> createUsers(final List<String> emails) {
-        return null;
+        Set<String> emailSet = new HashSet<>(emails);
+        return emailSet.stream().
+                filter(Objects::nonNull).map(User::new)
```

Comment:
Nice to have: let's better stick to single approach and always keep the dot `.` together with the method name.
I mean this:
```
collection.stream()
  .filter(..)
  .map(..)
```
is always better than this:
```
collection.stream.
  filter(..)
  .map(..)
```